@navikt/core/react/src/list/ListItem.tsx

```diff
@@ -0,0 +1,72 @@
+import cl from "clsx";
+import React, { forwardRef } from "react";
+import { BodyShort } from "../typography";
+
+export interface ListItemProps extends React.HTMLAttributes<HTMLLIElement> {
+  /**
+   * Content for description of the list item
+   */
+  children: React.ReactNode;
+  /**
+   * Title for the list item
+   */
+  title?: string;
+  /**
+   * Allows setting a different HTML h-tag
+   * @default "h4"
+   */
+  headingTag?: React.ElementType<any>;
+  /*
+   * Icon to be used as list marker
+   */
+  icon?: React.ReactNode;
+}
+
+export interface ListItemType
+  extends React.ForwardRefExoticComponent<
+    ListItemProps & React.RefAttributes<HTMLLIElement>
+  > {}
+
+export const ListItem: ListItemType = forwardRef(
+  ({ className, children, title, headingTag = "h4", icon, ...rest }, ref) => {
+    return (
+      <li {...rest} ref={ref} className={cl("navds-list__item", className)}>
+        <div className={`navds-list__item__icon${!icon && "--default"}`}>
+          {icon ? icon : <ListMarker />}
+        </div>
+
+        <div className="navds-list__item__content">
+          {title && (
+            <BodyShort
+              as={headingTag}
+              size="small"
+              className="navds-list__item__title"
```

Comment:
```suggestion
              className="navds-list__item-title"
```

Vet ikke hva som er mest "riktig" i BEM-verden, men har brukt konvensjonen med `-` separator mellom flere sub-elementer for det meste i kodebasen tror jeg